src/AppBundle/Entity/Dashboard.php

```diff
@@ -0,0 +1,142 @@
+<?php
```

Comment:
Please check to add additional validations, for example NotNull or NotBlank and so on